.github/workflows/iroha2-dev-pr.yml

```diff
@@ -19,12 +19,14 @@ jobs:
         with:
           path: ${{ env.CACHE_PATHS }}
           key: ${{ hashFiles('Cargo.toml') }}
+      - name: Install cargo-lints
+        run: cargo install --git https://github.com/i1i1/cargo-lints
       - name: Format check
         run: cargo fmt --all --verbose -- --check
       - name: Static analysis without features
-        run: cargo clippy --workspace --benches --tests -- -Dwarnings
+        run: cargo-lints clippy --workspace --benches --tests -- -Dwarnings
```

Comment:
Should it be `cargo lint clippy...` ?

Reply:
Yeah, was struggling with enabling it to work